### DISCUSSION TITLE
How to use multiple cli instance for same mobile (username)

### DISCUSSION THREAD

**Discussion by Devarsh-leo** [AsamK/signal-cli#527] [Q&A]
What I want to do is that I want to get data again which I received while i was testing signal-cli. I thought to make new signal-cli instance but I don't know how to make one.

**Comment by bublath** [comment]
Not exactly sure what you want to achieve. Are you talking about the dbus daemon or the client?
Basically you cannot have two instances running using the same profile (it gets locked).
You can run a second instance using a copy of the profile (--config to a different location) but when I did this once (testing on two system) basically one got disabled (its a secured connection after all and after the new one exchanges keys with the server, the old one becomes invalid).
However what you can do, is you can link a device. It will get the same data as the "master". So far I however believe that the only way to  actually allow the linking is take a picture of the qrcode - and that is not supported by signal-cli. So you probably should link your mobile phone number to signal-cli (even on multiple instances) to achieve parallel message processing.
